src/components/patient/PatientDetails.tsx

```diff
@@ -0,0 +1,80 @@
+import { Breadcrumbs, Container, Grid, Link, Typography } from '@mui/material'
+import { useMatch } from '@tanstack/react-location'
+import { useQuery } from '@tanstack/react-query'
+import { AxiosError } from 'axios'
+import { useAppConfig } from '../../hooks/useAppConfig'
+import ApiClient from '../../services/ApiClient'
+import PatientRecord from '../../types/PatientRecord'
+import Loading from '../common/Loading'
+import ApiErrorMessage from '../error/ApiErrorMessage'
+import NotFound from '../error/NotFound'
+import AddressPanel from './AddressPanel'
+import DemographicsPanel from './DemographicsPanel'
+import IdentifiersPanel from './IdentifiersPanel'
+import RegisteringFacilityPanel from './RegisteringFacilityPanel'
+import RelationshipPanel from './RelationshipPanel'
+import SubHeading from './SubHeading'
+
+const PatientDetails = () => {
+  const {
+    data: { uid }
+  } = useMatch()
+  const { getPatientName } = useAppConfig()
+  const { data, error, isLoading, isError } = useQuery<
+    PatientRecord,
+    AxiosError
+  >({
+    queryKey: ['patient', uid],
+    queryFn: async () => await ApiClient.getPatient(uid as string),
+    refetchOnWindowFocus: false
+  })
+
+  if (isLoading) {
+    return <Loading />
+  }
+
+  if (isError) {
+    return <ApiErrorMessage error={error} />
+  }
+
+  if (!data) {
+    return <NotFound />
+  }
+
+  const patientName = getPatientName(data)
+
+  return (
+    <Container>
```

Comment:
```suggestion
    <Container maxWidth="xl">
```
Lots of data so just a little breathing room